Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -284,28 +305,52 @@ private String applyRegex(String text) {
         }
         return text;
     }
-    
+
     private static Boolean IsMocked() {
-        // TODO: support un mocked mode
+        String keyValue=System.getProperty("isMockedCase");
```

Comment:
I see a lot of space issue here, suggest using checkstyle tool and correct them.